server/methods/core/groups.js

```diff
@@ -24,9 +24,9 @@ Meteor.methods({
     check(groupData, Object);
     check(groupData.name, String);
     check(groupData.description, Match.Optional(String));
-    check(groupData.permissions, [String]);
+    check(groupData.permissions,  Match.Optional([String]));
     check(shopId, String);
-    let res;
+    let _id;
 
     if (!Reaction.hasPermission("admin")) {
```

Comment:
As we chatte about, let's make sure these are checking permissions for the shopId that is associated with the group being modified.